### ISSUE TITLE
image composite masked nodes

### ISSUE THREAD

**Issue by mliand** [WASasquatch/was-node-suite-comfyui#264]
Hello author, I recently tried to use animatediff to create animations. When using the imagecompositemask node, I can only output one picture. I input a 16-frame picture. After combining it with the mask, the output is only one. I hope it can support multiple frames. Image input and output

**Comment by WASasquatch** [comment]
Yeah a lot of those nodes are really old before I added any compatible batch support. I don't have the time to really go through this right now, but am open to PRs.
